presto-spi/src/main/java/com/facebook/presto/spi/ColumnHandle.java

```diff
@@ -13,6 +13,17 @@
  */
 package com.facebook.presto.spi;
 
+import static com.facebook.presto.spi.StandardErrorCode.NOT_SUPPORTED;
+
 public interface ColumnHandle
 {
+    default ColumnIdentity getColumnIdentity()
```

Comment:
I'm not sure this should be in the ColumnHandle. If the connector needs additional state to turn a column handle in to a column identity, it would require ColumnHandle to keep an internal reference to ConnectorMetadata.